src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -63,29 +64,39 @@ public ApplicationExtract(ILogger<ApplicationExtract> log, ApplyDataContext appl
             }
 
             _logger.LogInformation($"ApplicationExtract function executed at: {DateTime.Now}");
+            
+            var currentApplicationId = Guid.Empty;
 
             try
             {
                 var applications = await GetApplicationsToExtract(DateTime.Now);
 
                 foreach (var applicationId in applications)
                 {
-                    var answers = await ExtractAnswersForApplication(applicationId);
+                    currentApplicationId = applicationId;
```

Comment:
_logger.LogInformation("Starting to extract application {applicationId}", applicationId);